CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Changed
+
+- `ForceFlush` of the `NewBatchSpanProcessor` in `go.opentelemetry.io/otel/sdk/trace` calls `SpanExporter.ExportSpans` synchronously. (#6416)
+
```

Comment:
Just for the record, this is a significant change in behavior, and can easily be interpreted as a breaking change.

Reply:
Kind of https://github.com/dwmkerr/hacker-laws#hyrums-law-the-law-of-implicit-interfaces, right?

Reply:
More like this https://xkcd.com/1172/